src/Physics/BeamHNL/FormFactorTables.h

```diff
@@ -0,0 +1,165 @@
+//----------------------------------------------------------------------------
+/*!
+
+  Lookup tables provided for the functions `f_{1}(x)` and `f_{2}(x)`
+  that enter the N --> \nu \ell_{\beta} \ell_{\beta} channel
+
+\namespace  genie::hnl::selector
+
+\brief      Form factor lookup tables
+
+\author     John Plows <komninos-john.plows@physics.ox.ac.uk>
+
+\created    January 25th, 2022
+
+\cpright    Copyright (c) 2003-2022, The GENIE Collaboration
+            For the full text of the license visit http://copyright.genie-mc.org
+
+*/
+//----------------------------------------------------------------------------
+
+#ifndef _HNL_JFORMFACTORTABLES_H_
+#define _HNL_JFORMFACTORTABLES_H_
+
+namespace genie {
+namespace hnl {
+
+    namespace selector {
+
+      // lookup tables, 50x10 + 1
+      static const double PARTWIDTH = 1e-3;
+      
+      static const double FormfactorX[] = { 0.000, 0.001, 0.002, 0.003, 0.004, 0.005, 0.006, 0.007, 0.008, 0.009, 0.010, 0.011, 0.012, 0.013, 0.014, 0.015, 0.016, 0.017, 0.018, 0.019, 
```

Comment:
Can we document here the origin of all these numbers. Do you expect these numbers to change, or to have a need for an “alternative” set of numbers. There is an easy way to specify such arrays of numbers in the XML configuration, if that is more suitable. Perhaps @mroda88 can comment

Reply:
These tables were computed using Eq. (3.52) of [_EPJ_ **C 81** (2021) 78](https://link.springer.com/article/10.1140/epjc/s10052-021-08861-y). They're consistent with the Lagrangian of that paper (which is what's implemented in this module). Ideally in the future we can have a way for the user to directly give us decay widths for HNL production / decay depending on mass / mixings through some XML configuration, but it is not yet clear to me what the optimal way forward is.